test/test.cli.ts

```diff
@@ -6,18 +6,8 @@ import * as util from 'util';
 import enableDestroy = require('server-destroy');
 import {LinkResult, LinkState} from '../src/index';
 
-// eslint-disable-next-line prefer-arrow-callback
-describe('cli', function () {
+describe('cli', () => {
   let server: http.Server;
-  this.timeout(60_000);
```

Comment:
I think the default timeout is not enough for the Windows CI :)

Reply:
Goodness gravy.